src/scripts/main.js

```diff
@@ -357,4 +357,41 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+function calcGender(gender) {
+  if (gender === 'm') {
+    return 'Male';
+  } else {
+    return 'Female';
+  }
+}
+
+function calcAge(personBorn, personDied) {
+  return personDied - personBorn;
+}
+
+function calcCentury(diedYear) {
+  return Math.ceil(diedYear / 100);
+}
+
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the HTML document. If it doesn't, `document.querySelector` will return `null`, and attempting to call `insertRow` on `null` will result in a runtime error.